test/Makefile

```diff
@@ -136,7 +136,7 @@ xmpparser-test:
 
 version_test \
 unit_test    \
-test         \
```

Comment:
The removed `test` target is still explicitly called in these two places at least:

https://github.com/Exiv2/exiv2/blob/dd2d181755a6e642c0a8e3225ef5407fff49eb3a/CMakeLists.txt#L99
https://github.com/Exiv2/exiv2/blob/d5c6391c5f622e1e61561789647d25d2b5563a10/test/Makefile#L45

Reply:
I don't see an issue with CMakeLists.txt, the current code is:

```cmake
if( EXIV2_BUILD_UNIT_TESTS )
    add_subdirectory ( unitTests )
    add_custom_target(unit_test
        COMMAND env EXIV2_BINDIR="${CMAKE_RUNTIME_OUTPUT_DIRECTORY}" make unit_test
        WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}/test"
    )
endif()
```

I agree that test/Makefile#45 should be:

```make
all: tests
```

I intend to invite @LeoHsiao1 to deal with #1269 and will add an explicit mention that test/Makefile#45 requires attention.

Reply:
That does not look like the current code on master. I linked the current version in the previous comment and the CMakeLists.txt still mention `make test` there.

Reply:
I don't maintain 'master'.  We had a team meeting at my home in England in May 2018.  We agreed that I would be the release engineer for v0.27 and subsequent v0.27 dot releases.  I back-port security features from branch 'master' to branch 0.27-maintenance.  I don't believe I have submitted a PR to 'master' since completing 0.27 in December 2018.  Regrettably there have been almost no changes to 'master' since January 2020.

Reply:
Oh, right, I did not notice this pull request targets 0.27 – I was confused by 0.27->master in the title.

Reply:
Ah yes.  (0.27->master) in the title is a note that this change is in 0.27 and should be merged into master.